README.md

```diff
@@ -52,6 +52,7 @@ Before installing mmcv-full, make sure that PyTorch has been successfully instal
 
 We provide pre-built mmcv packages (recommended) with different PyTorch and CUDA versions to simplify the building.
 
+**For pip < 20.3 and mmcv <= 1.2.1, please refer to the following table:**
```

Comment:
Is `mmcv <= 1.2.1` a necessary limitation?

Reply:
nope... let me revise it